ui/src/components/Menu.svelte

```diff
@@ -97,13 +96,14 @@
         SendNUI("clearBlips");
       }}
     >
-      <i class="fas fa-ban text-[1.5vh]"></i>
+    <i class="fas fa-ban text-[1.5vh]"></i>
+
     </button>
   </div>
   <!-- MENU -->
   <div class="w-[25%] h-[90%] mr-[4vh] overflow-auto pr-[0.5vh]">
     {#if $DISPATCH_MENU}
-    {#each $DISPATCH_MENU.filter(dispatch => dispatch.message && dispatch.jobs.includes($PLAYER.job.type)).slice().reverse() as dispatch}
+    {#each $DISPATCH_MENU.slice(-$MAX_CALL_LIST).filter(dispatch => dispatch.message && dispatch.jobs.includes($PLAYER.job.type)).slice().reverse() as dispatch}
```

Comment:
Please extract this to the JavaScript. It is quite hard to figure out what is going on here. This includes not just copy pasting it into a variable but explaining what is going on when altering the array.

Reply:
thats the solution i had from okiez and its working fantastic, i dont know how to make that for js